### PR TITLE
Fix/bug map too small

### PR DESCRIPTION
**Pull request by davidholzwarth**
# Summary

Whenever scrolling downwards navigating to the south with the map is not shown on the edge of the BottomScaffold. To fix this, I reduced the size of the bottomSheet padding in all screens that have a map, so that the map displays even under the section of the bottomSheet that does not cover the complete width of the screen as a result of its rounded corners.

### CODE REVIEW

File: app/src/main/java/ch/hikemate/app/ui/map/HikeDetailsScreen.kt [hunk 1]
```diff
@@ -272,7 +272,11 @@ fun hikeDetailsMap(hike: DetailedHike): MapView {
       factory = { mapView },
       modifier =
           Modifier.fillMaxWidth()
-              .padding(bottom = 300.dp) // Reserve space for the scaffold at the bottom
+              .padding(
+                  bottom =
+                      300.dp -
+                          20.dp) // Reserve space for the scaffold at the bottom, -20.dp to avoid
+              // the map being to small under the bottomSheet
```
Comment (MaelImhof) [inline app/src/main/java/ch/hikemate/app/ui/map/HikeDetailsScreen.kt:279]: Nitpick: maybe this is a good opportunity to refactor the hike details screen and put those values in constants in the `HikeDetailsScreen` object. The constants should not be shared between screens, as we might want to make the bottom sheet slightly higher on details screen than on map screen for example.

File: app/src/main/java/ch/hikemate/app/ui/map/MapScreen.kt [hunk 2]
```diff
@@ -446,7 +446,11 @@ fun MapScreen(
                     modifier =
                         Modifier.fillMaxSize()
                             .testTag(MapScreen.TEST_TAG_MAP)
-                            .padding(bottom = MapScreen.BOTTOM_SHEET_SCAFFOLD_MID_HEIGHT))
+                            .padding(
+                                bottom =
+                                    MapScreen.BOTTOM_SHEET_SCAFFOLD_MID_HEIGHT -
+                                        20.dp)) // -20.dp to avoid the map being to small under
```
Comment (MaelImhof) [inline app/src/main/java/ch/hikemate/app/ui/map/MapScreen.kt:452]: Nitpick: define this value in a constant in the `MapScreen` object. The constants should not be shared between screens, as we might want to make the bottom sheet slightly higher on details screen than on map screen for example.

File: app/src/main/java/ch/hikemate/app/ui/map/RunHikeScreen.kt [hunk 3]
```diff
@@ -184,7 +184,11 @@ private fun runHikeMap(hike: DetailedHike): MapView {
       factory = { mapView },
       modifier =
           Modifier.fillMaxWidth()
-              .padding(bottom = 300.dp) // Reserve space for the scaffold at the bottom
+              .padding(
+                  bottom =
+                      300.dp -
+                          20.dp) // Reserve space for the scaffold at the bottom, -20.dp to avoid
+              // the map being to small under the bottomSheet
```
Comment (MaelImhof) [inline app/src/main/java/ch/hikemate/app/ui/map/RunHikeScreen.kt:191]: Nitpick: maybe this is a good opportunity to refactor the run hike screen and put those values in constants in the `RunHikeScreen` object. The constants should not be shared between screens, as we might want to make the bottom sheet slightly higher on details screen than on map screen for example.

**Review by MaelImhof — CHANGES_REQUESTED** [review]
Nice and clean fix ! Just nitpicking to avoid magic numbers.

File: app/src/main/java/ch/hikemate/app/ui/map/MapScreen.kt (hunk 2 above)
Comment (davidholzwarth) [reply app/src/main/java/ch/hikemate/app/ui/map/MapScreen.kt:452]: What do you mean? You mean add 20.dp as a constant in the MapScreen object?

**Comment by MaelImhof** [comment]
I believe a test was broken by the new padding, could you take a look at the CI output?

**Review by MaelImhof — APPROVED** [review]
Still looks good to me 👌🏻

**Pull request merged**